drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -1,4 +1,11 @@
 matterManufacturer:
+#ThirdReality
+  - id: "ThirdReality/WiFi"
+    deviceLabel: THIRDREALITY NL
```

Comment:
I think the device label would be improved by writing out "NL" to "Night Light".

Reply:
Done!  Please review, thanks!